multimod/cmd/prerelease.go

```diff
@@ -55,7 +54,7 @@ var prereleaseCmd = &cobra.Command{
 		}
 	},
 	Run: func(*cobra.Command, []string) {
-		fmt.Println("Using versioning file", versioningFile)
+		log.Println("Using versioning file", versioningFile)
```

Comment:
Doesn't this add the logging prefix to this message? Should `Fprint` be used instead?

Reply:
It does not: 
https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/5655634a2a97cda818cba89b0fdf95b6cfe3b9ee/multimod/cmd/tag.go#L48-L49